src/components/menus/ApiKeyManager.ts

```
@@ -0,0 +1,362 @@
+
+/**
+ * ApiKeyManager component for the toolbar
+ * This component allows users to manage their API keys for different AI providers
+ */
+export class ApiKeyManager extends AbstractMenuButton {
+    private dialog: HTMLDivElement | null = null;
+    private mask: HTMLDivElement | null = null;
+
+    constructor() {
+        super();
+    }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)